aspnetcore/blazor/index.md

```diff
@@ -17,25 +17,30 @@ By [Daniel Roth](https://github.com/danroth27) and [Luke Latham](https://github.
 Blazor is a framework for building interactive client-side web UI with .NET:
 
 * Create rich interactive UIs using C# instead of JavaScript.
-* Share server-side and client-side app logic written with .NET.
+* Share server-side and client-side app logic written in .NET.
 * Render the UI as HTML and CSS for wide browser support, including mobile browsers.
 
 Using .NET for client-side web development offers the following advantages:
 
 * Write code in C# instead of JavaScript.
 * Leverage the existing .NET ecosystem of .NET libraries.
-* Share app logic across the server and client.
+* Share app logic across server and client.
 * Benefit from .NET's performance, reliability, and security.
 * Stay productive with Visual Studio on Windows, Linux, and macOS.
 * Build on a common set of languages, frameworks, and tools that are stable, feature-rich, and easy to use.
 
 ## Components
 
-Blazor apps are based on *components*. A component in Blazor is an element of UI, such as a page, dialog, or data entry form. Components handle user events and define flexible UI rendering logic. Components can be nested and reused.
+Blazor apps are based on *components*. A component in Blazor is an element of UI, such as a page, dialog, or data entry form.
 
-Components are .NET classes built into .NET assemblies that can be shared and distributed as [NuGet packages](/nuget/what-is-nuget). The component class is usually written in the form of a Razor markup page with a *.razor* file extension.
+Components are .NET classes built into .NET assemblies that:
 
-Components in Blazor are sometimes referred to as *Razor components*. [Razor](xref:mvc/views/razor) is a syntax for combining HTML markup with C# code designed for developer productivity. Razor allows you to switch between HTML markup and C# in the same file with [IntelliSense](/visualstudio/ide/using-intellisense) support. Razor Pages and MVC also use Razor. Unlike Razor Pages and MVC, which are built around a request/response model, components are used specifically for client-side UI logic and composition.
+* Define flexible UI rendering logic.
+* Handle user events.
+* Can be nested and reused.
+* Can be shared and distributed as [Razor class libraries](xref:razor-pages/ui-class) or [NuGet packages](/nuget/what-is-nuget).
+
+The component class is usually written in the form of a [Razor](xref:mvc/views/razor) markup page with a *.razor* file extension. Components in Blazor are sometimes referred to as *Razor components*. Razor is a syntax for combining HTML markup with C# code designed for developer productivity. Razor allows you to switch between HTML markup and C# in the same file with [IntelliSense](/visualstudio/ide/using-intellisense) support. Razor Pages and MVC also use Razor. Unlike Razor Pages and MVC, which are built around a request/response model, components are used specifically for client-side UI logic and composition.
```

Comment:
Should the *Razor components* reference be replaced with Blazor server-side?

Reply:
Not here ... the point is to say that sometimes components are called "Razor components."

Reply:
Alternatively to this, we could say ...

> Components in Blazor are formally referred to as *Razor components*.

I kind'a like that better actually.

Reply:
@danroth27 I don't think u need to review this PR. The vast majority of updates are nits we've already discussed. On this point tho :point_up:, how would u like to clarify the text?

Reply:
> Components in Blazor are formally referred to as Razor components.

I like this better too.